File: src/main/java/com/manson/domain/config/ArmorConfig.java
package com.manson.domain.config;

import com.manson.domain.AbstractObject;
import com.manson.domain.fo76.items.enums.ArmorGrade;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@EqualsAndHashCode(callSuper = false)
@ToString(callSuper = true)
public class ArmorConfig extends AbstractObject {

    private String helper;
    private int dr;
    private int er;
    private int rr;
    private String shortTerm;
    private String armorType;
    private String armorPart;
    private ArmorGrade armorGrade = ArmorGrade.Unknown;
    private String armorId;
    private String gradeId;

}

File: src/main/kotlin/com/manson/domain/config/ArmorConfig.kt
package com.manson.domain.config

import com.manson.domain.fo76.items.enums.ArmorGrade

open class ArmorConfig {
    var helper: String = ""
    var dr = 0
    var er = 0
    var rr = 0
    var shortTerm: String = ""
    var armorType: String = ""
    var armorPart: String = ""
    var armorGrade: ArmorGrade = ArmorGrade.Unknown
    var armorId: String = ""
    var gradeId: String = ""
}